main.py

```diff
@@ -79,13 +82,17 @@ def create_overlays(ground, ego):
 # predefine actions on key: keypress-states connected to actions that will be performed if state is given
 color_key_indicator_blue = lambda: key_indicator.set_color(colors_dict["blue"])
 color_key_indicator_cyan = lambda: key_indicator.set_color(colors_dict["cyan"])
+toggle_overlays = lambda overlays: [overlay.toggle_visibility() for overlay in overlays]
 
 act_change_box_color_on_spacebar_pressed = LandingGameActionOnKey(
     PygameKeyState(pygame.K_SPACE, True), color_key_indicator_blue
 )
 act_change_box_color_on_spacebar_not_pressed = LandingGameActionOnKey(
     PygameKeyState(pygame.K_SPACE, False), color_key_indicator_cyan
 )
+act_toggle_overlays_on_v_pressed = LandingGameActionOnKey(
+    PygameKeyState(pygame.K_v, True), lambda: toggle_overlays(overlays)
+)
```

Comment:
Is it better to merge it like this or wait until the underlying bug is fixed and adjust the code of this pr before merging?